it-and-security/teams/virtual-machines.yml

```
@@ -0,0 +1,15 @@
+name: "Virtual machines"
+team_settings:
+  features:
+    enable_host_users: true
+    enable_software_inventory: true
+  host_expiry_settings:
+    host_expiry_enabled: false
+    host_expiry_window: 0
+  secrets:
+    - secret: $DOGFOOD_VIRTUAL_MACHINES_ENROLL_SECRET
+agent_options:
+  path: ../lib/virtual-machines.agent-options.yml
```

```suggestion
  path: ../lib/agent-options.yml
```

Propose using the same agent options as other teams for now. Until we want the VMs team to have it's own options.

Why? Less to maintain.